### DISCUSSION TITLE
Home Page is empty

### DISCUSSION THREAD

**Discussion by ghost** [Ylianst/MeshCentral#6020] [General]
Hi,

The home page of my server is empty for some reason.
Here I have to 2 questions:
1. Can I customize the home page to my liking?
2. Can I set a specific page to be the home page e.g. My Devices as home page?
Thanks.
![EmptyHomePage](https://github.com/Ylianst/MeshCentral/assets/102313998/cdac334e-a2cc-4df0-9c97-3d8e46a0d0f2)

Thanks.

**Comment by ghost** [comment]
Hi, sorry for asking, is there any way to customize the homepage or set the default home page on Devices?
Thanks.

**Reply by VINISHVV** [reply]
Do you want to customize the login screen / default page showing after user logged in?

**Reply by ghost** [reply]
If I could do something in the homepage would be great.
If I can set the home page to be the Device page it would be also great.

**Reply by si458** [reply]
Im more confused why ur devices page is white, that looks like a bug?

But anyways Customising the home page is very technical and advanced, you have to copy pages over then edit the source code, then translate and minify. And at every meshcentral update u have to copy the files over again and reapply ur customising

Also you can't specify a default page to visit when u login
(You would need to create a github issues enhancement request for this to be added at somepoint)

**Reply by ghost** [reply]
No my devices page is not white.
I need to press the first icon on lcon list on the left to see the device page.
When I log in I see this  and then, I need to press the Devices icon on the left to get to the Devices page.
But now it doesn't happen like it seems that the home page is back to be the default Devices page.
